examples/rules.mk

```diff
@@ -2,20 +2,26 @@
 # Common make rules for all examples
 # ---------------------------------------
 
-ifeq ($(CROSS_COMPILE),xtensa-esp32s2-elf-)
-# Espressif IDF use CMake build system, this add wrapper target to call idf.py
+ifeq ($(findstring esp32s, $(BOARD)), esp32s)
+$(info Vendor: $(VENDOR), family: $(CHIP_FAMILY), target: $(CHIP_TARGET))
```

Comment:
not all board has esp32s2 or esp32s3 in their name, we should just use the CHIP_FAMILY. It is good enough.

Reply:
The commit I was based on contains the names of the espressif boards started with target prefix. This looked useful for me. This was updated accordingly as it is in latest master.

Reply:
name like Adafruit funhouse or tinyS2 etc. won't help esp32s2 or s3. It will cause issue with other boards in the future. The $(VENDOR) variable will probably be removed as well. I don't see the use of it in the make. Board name is already uniquely defined.  If you think `esp32s2_saola_1` does not reflect the espressif as vendor, we could rename it to `espressif_saola_1`

Reply:
Ah, I see now. The board name is enough to determine all required information as `$(CHIP_TARGET), $(FAMILY)` and so on...